src/main/java/org/jitsi/jicofo/jigasi/JigasiDetector.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Jicofo, the Jitsi Conference Focus.
+ *
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.jicofo.jigasi;
+
+import net.java.sip.communicator.impl.protocol.jabber.extensions.colibri.*;
+import net.java.sip.communicator.util.*;
+import org.jitsi.jicofo.*;
+import org.jitsi.jicofo.xmpp.*;
+import org.jitsi.service.configuration.*;
+
+/**
+ * @author Damian Minkov
```

Comment:
Some high level description is missing. What does it do what config options activate it ?